### DISCUSSION TITLE
Cannot create Home Assistant VM in Proxmox 8.1.3 arm64 port

### DISCUSSION THREAD

**Discussion by elterko** [tteck/Proxmox#2555] [Help Needed] [answered]
Greetings, tried to create a HA VM using the Proxmox amr64 port from https://github.com/jiangcuo/Proxmox-Port and failed when creating HAOS VM per the following error:
` Created HAOS VM (haos11.5)
 - Starting Home Assistant OS VM...bridge 'vmbr0' does not exist
kvm: -netdev type=tap,id=net0,ifname=tap100i0,script=/var/lib/qemu-server/pve-bridge,downscript=/var/lib/qemu-server/pve-bridgedown,vhost=on: network script /var/lib/qemu-server/pve-bridge failed with status 512
start failed: QEMU exited with code 1
‼ ERROR 255@332 Unknown failure occurred.`

vmbr0 was manually created in Proxmox UI prior to executing your script

**Comment by tteck** [comment]
Which script did you use?

**Reply by elterko** [reply]
Pimox

**Reply by tteck** [reply]
yeah I get that, which Home Assistant OS VM script did you use?

**Reply by elterko** [reply]
pimox-haos-vm.sh, the one I pulled from https://tteck.github.io/Proxmox/ under "Pimox", all default settings

**Reply by tteck** [reply, marked as answer]
> bridge 'vmbr0' does not exist

Looks like you have a "custom" setup. Use Advanced settings to match your setup.

**Reply by elterko** [reply]
I wished I had a custom setup, all stock, Raspberry PI OS Lite 64bit with Proxmox arm64 port installed following original instructions from https://github.com/jiangcuo/Proxmox-Port

**Reply by tteck** [reply]
> bridge 'vmbr0' does not exist

I'm talking about settings

**Reply by elterko** [reply]
/etc/network/interfaces is completely empty

**Reply by tteck** [reply]
Try re-installing Pimox 🤷🏻‍♂️ https://github.com/pimox/pimox7
I have no ~toy~ hardware to test
